README.md

```diff
@@ -0,0 +1,10 @@
+# l2switch
+This branch is a mirror of the OpenDaylight l2switch gerrit project
+
+**Purposes:** Branch is intended to be tested with x86 architectuce enivronments like CloudLab VMs used in our [demo](https://github.com/slab14/IoT_Sec_Gateway).
+
+**How it works:**
+
```

Comment:
Maybe a little description here?
- build with build.sh
- run with startODL.sh
- SDN controller, assumes a virtual switch is running locally named br0, listening on port 6677. Additionally it assumes that docker is running and listening on port 4243. 

Also add a note that it assumes the use of my branch of ovs. for the ovs-docker-remote command.